src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1688,15 +1629,18 @@ key is not present, no replacement occurs):
 </task>
 ----
 
-== Scheduler and DataSpace APIs
-From a ProActive task, several APIs are accessible via Java objects bound to dedicated variables:
+[[_task_apis]]
+== Scheduler, DataSpace, and Synchronization APIs
+
+From a <<_glossary_task_executable_script,Script Task>>, several APIs are accessible via Java objects bound to dedicated <<_glossary_script_bindings,script bindings>>:
 
-- Scheduler API : using the "schedulerapi" variable, it is possible to interact directly with the scheduler and do several operations such as submitting jobs, waiting for tasks, listing running jobs etc.
-- DataSpace API : the "userspaceuri" or "globalspaceuri" variables allow to manipulate files present in User or Global spaces. Operations include listing, uploading, downloading, or deleting files.
+- Scheduler API : using the `schedulerapi` binding, it is possible to interact directly with the scheduler and do several operations such as submitting jobs, waiting for tasks, listing running jobs etc.
```

Comment:
I would remove the `-ing` suffix from the actions list

Reply:
too many ing... Ok I will try to improve this